src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -412,6 +425,17 @@ export async function generateFeaturesConfig(params: { extensionPath: string; cw
 	return featuresConfig;
 }
 
+async function prepareGHCRauthAndOCICache(dstFolder: string) {
```

Comment:
should this function be renamed and my POC comments removed?

Reply:
Fixed.